src/main/java/org/dependencytrack/util/InternalComponentIdentificationUtil.java

```diff
@@ -35,40 +29,8 @@ public final class InternalComponentIdentificationUtil {
     private InternalComponentIdentificationUtil() {
     }
 
-    public static boolean isInternalComponent(final Component component, final QueryManager qm) {
-        return isInternalGroup(component.getGroup(), qm) || isInternalName(component.getName(), qm);
-    }
-
-    private static boolean isInternalGroup(final String group, final QueryManager qm) {
-        if (StringUtils.trimToNull(group) == null) {
-            return false;
-        }
-
-        final ConfigProperty internalGroupsRegexProperty = qm.getConfigProperty(
-                ConfigPropertyConstants.INTERNAL_COMPONENTS_GROUPS_REGEX.getGroupName(),
-                ConfigPropertyConstants.INTERNAL_COMPONENTS_GROUPS_REGEX.getPropertyName()
-        );
-        if (internalGroupsRegexProperty == null || StringUtils.trimToNull(internalGroupsRegexProperty.getPropertyValue()) == null) {
-            return false;
-        }
-
-        return Pattern.matches(StringUtils.trimToNull(internalGroupsRegexProperty.getPropertyValue()), group);
-    }
-
-    private static boolean isInternalName(final String name, final QueryManager qm) {
-        if (StringUtils.trimToNull(name) == null) {
-            return false;
-        }
-
-        final ConfigProperty internalNamesRegexProperty = qm.getConfigProperty(
-                ConfigPropertyConstants.INTERNAL_COMPONENTS_NAMES_REGEX.getGroupName(),
-                ConfigPropertyConstants.INTERNAL_COMPONENTS_NAMES_REGEX.getPropertyName()
-        );
-        if (internalNamesRegexProperty == null || StringUtils.trimToNull(internalNamesRegexProperty.getPropertyValue()) == null) {
-            return false;
-        }
-
-        return Pattern.matches(StringUtils.trimToNull(internalNamesRegexProperty.getPropertyValue()), name);
+    public static boolean isInternalComponent(final Component component) {
```

Comment:
Do we need this util class? We can use `InternalComponentIdentifier().isInternal` directly instead?

Reply:
Kept it mostly for backward-compatibility. But you're right, it doesn't really serve any real purpose. Let me remove it.

Reply:
Removed!